templates/telco/openran.html

```diff
@@ -0,0 +1,490 @@
+
+{% extends "templates/one-column.html" %}
+
+{% block title %}OpenRAN{% endblock %}
+
+{% block meta_description %}OpenRAN - disaggregated, modular and open source radio networks{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/181cqeYzSZ_RinjPTMMNDyAEMwKVvdeoc30iQDrwl4aQ/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip is-bordered u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7">
+        <h1>
+          Open source in the radio network
+        </h1>
+        <p class="p-heading--3">
+          The biggest game changer in the telecom space
+        </p>
+        <p>
+          The disaggregation of RAN HW and SW speeds up innovation and allows true multi-vendor, flexible deployment based on open interfaces.
+        </p>
+        <p>
+          <a href="/telco/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+        </p>
+      </div>
+      <div class="col-5 u-align--center u-hide--small">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/d2387483-telecommunications-grey.svg",
+            alt="",
+            width="308",
+            height="300",
+            hi_def=True,
+            loading="lazy"
```

Comment:
I think as this is the first image, you should maybe load it immediately instead of doing lazy loading